worlds/civ_6/Items.py

```diff
@@ -0,0 +1,296 @@
+    "CIVIC_MERCANTILISM",
+    "CIVIC_CONSERVATION",
+    "CIVIC_CIVIL_SERVICE",
+    "CIVIC_GLOBALIZATION",
+    "CIVIC_COLD_WAR",
+    "CIVIC_URBANIZATION",
+    "CIVIC_NATIONALISM",
+    "CIVIC_MOBILIZATION",
+    "PROGRESSIVE_NEIGHBORHOOD",
+    "PROGRESSIVE_PRESERVE"
+]
+
+
+class FillerItemRarity(Enum):
+    COMMON = "COMMON"
+    UNCOMMON = "UNCOMMON"
+    RARE = "RARE"
+
+
+FILLER_DISTRIBUTION: Dict[FillerItemRarity, float] = {
+    FillerItemRarity.RARE: 0.025,
+    FillerItemRarity.UNCOMMON: .2,
+    FillerItemRarity.COMMON: 0.775,
+}
+
+
+class FillerItemData:
+    name: str
+    type: str
+    rarity: FillerItemRarity
+    civ_name: str
+
+    def __init__(self, data: Dict[str, str]):
+        self.name = data["Name"]
+        self.rarity = FillerItemRarity(data["Rarity"])
+        self.civ_name = data["Type"]
+
+
+def get_filler_item_data() -> Dict[str, FillerItemData]:
+    """
+    Returns a dictionary of filler items with their data
+    """
+    goody_huts: List[Dict[str, str]] = get_goody_hut_rewards_data()
+    # Create a FillerItemData object for each item
+    cached_filler_items = {item["Name"]: FillerItemData(item) for item in goody_huts}
+
+    return cached_filler_items
+
+
+class CivVIItemData:
+    civ_vi_id: int
+    classification: ItemClassification
+    name: str
+    code: int
+    cost: int
+    item_type: CivVICheckType
+    progression_name: Optional[str]
+    civ_name: Optional[str]
+
+    def __init__(self, name, civ_vi_id: int, cost: int, item_type: CivVICheckType, id_offset: int, classification: ItemClassification, progression_name: Optional[str], civ_name: Optional[str] = None):
+        self.classification = classification
+        self.civ_vi_id = civ_vi_id
+        self.name = name
+        self.code = civ_vi_id + CIV_VI_AP_ITEM_ID_BASE + id_offset
+        self.cost = cost
+        self.item_type = item_type
+        self.progression_name = progression_name
+        self.civ_name = civ_name
+
+
+class CivVIItem(Item):
+    game: str = "Civilization VI"
+    civ_vi_id: int
+    item_type: CivVICheckType
+
+    def __init__(self, item: CivVIItemData, player: int, classification: ItemClassification = None):
+        super().__init__(item.name, classification or item.classification, item.code, player)
+        self.civ_vi_id = item.civ_vi_id
+        self.item_type = item.item_type
+
+
+def format_item_name(name: str) -> str:
+    name_parts = name.split("_")
+    return " ".join([part.capitalize() for part in name_parts])
+
+
+def get_item_by_civ_name(item_name: List[str], item_table: Dict[str, 'CivVIItemData']) -> 'CivVIItemData':
+    """Gets the names of the items in the item_table"""
+    for item in item_table.values():
+        if item_name == item.civ_name:
+            return item
+
+    raise Exception(f"Item {item_name} not found in item_table")
+
+
+def _generate_tech_items(id_base: int, required_items: List[str], progressive_items: Dict[str, str]) -> List[CivVIItemData]:

```

Comment:
```suggestion
def _generate_tech_items(id_base: int, required_items: List[str], progressive_items: Dict[str, str]) -> Dict[str, CivVIItemData]:
```
Return type doesn't match what's being returned, and it's like this for most of the functions like this

Reply:
Ah nice catch